federatedgroups/lib/FederatedGroupShareProvider.php

```diff
@@ -114,18 +117,18 @@ public function __construct(
 		 $userManager
 		);
 		$this->groupManager = $groupManager;
-    	$this->federatedProvider = new FederatedShareProvider(
-			$dbConnection,
-			$eventDispatcher,
-			$addressHandler,
-			$notifications,
-			$tokenHandler,
-			$l10n,
-			$logger,
-			$rootFolder,
-			$config,
-			$userManager
-		);
+    	// $this->federatedProvider = new FederatedShareProvider(
```

Comment:
it is better to remove commented codes. we can refer to earlier commits to retrieve the original code